packages/swapper/src/swappers/ThorchainSwapper/utils/getLatestThorTxStatusMessage.ts

```diff
@@ -61,15 +61,18 @@ export const getLatestThorTxStatusMessage = (
           }
         }
 
-        return { message: obj.pending ? 'Swap pending' : 'Swap complete', status: TxStatus.Pending }
+        return {
+          message: obj.pending ? 'Swap pending' : 'Swap complete, awaiting finalization',
```

Comment:
Not for this PR at all, but it occurred to me whilst reading this that all of these probably need actual translations!